common/testing/testlogger/testlogger.go

```diff
@@ -472,15 +546,30 @@ func failureMessage(level string, msg string, tags []tag.Tag) string {
 	return b.String()
 }
 
-func errorMessage(level string, msg string) string {
+func errorMessage(level Level, msg string) string {
 	var b strings.Builder
 	b.WriteString("Unexpected ")
-	b.WriteString(level)
+	b.WriteString(level.String())
 	b.WriteString(" log encountered: ")
 	b.WriteString(msg)
 	return b.String()
```

Comment:
Somewhere around here must be a missing call to `String()` on a string builder; I'm seeing this after I threw in a `wh.logger.Panic("boom")` into our workflow handler:
```
 workflow_handler.go:2111: FAIL: Unexpected Panic log encountered: boomhost=127.0.0.1:62316 service=frontend 
        {%!s(*strings.Builder=&{0xc0003e5900 [47 85 115 101 114 115 47 115 116 101 112 104 97 110 47 87 111 114 107 115 112 97 99 101 47 100 101 102 97 117 108 116 47 116 101 109 112 111 114 97 108 47 99 111 109 109 111 110 47 116 101 115 116 105 110 103 47 116 101 115 116 108 111 103 103 101 114 47 116 101 115 116 108 111 103 103 101 114 46 103 111 58 52 53 56 32 103 111 46 116 101 109 112 111 114
```

Reply:
Yeah that's coming from failTest; I just fixed it internally